lib/player.js

```diff
@@ -6570,6 +6535,37 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return config;
   }
 
+  /**
+   * @return {!Object}
+   * @private
+   */
+  defaultLowLatencyConfig_() {
+    return {
+      streaming: {
+        inaccurateManifestTolerance: 0,
+        segmentPrefetchLimit: 2,
+        updateIntervalSeconds: 0.1,
+        maxDisabledTime: 1,
+        retryParameters: {
+          baseDelay: 100,
+        },
+      },
+      manifest: {
+        dash: {
+          autoCorrectDrift: false,
+        },
+        retryParameters: {
+          baseDelay: 100,
+        },
+      },
+      drm: {
+        retryParameters: {
+          baseDelay: 100,
+        },
```

Comment:
As this is plain object, keys can be minified by Closure. You should wrap all keys with `''`.

Reply:
I have tested in compiled mode and it works correctly!